apps/marketplace/components/SignupForm/SignupForm.js

```diff
@@ -1,371 +1,299 @@
 /* eslint-disable camelcase */
 import React from 'react'
 import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
 import { Form } from 'react-redux-form'
 import PageAlert from '@gov.au/page-alerts'
 
 import { required, validEmail, governmentEmail } from '../../components/validators'
 import Layout from '../../components/shared/Layout'
```

Comment:
we shouldn't put all the extra blank lines in the imports:
- absolute
- blank line
- relative
- blank line
- css modules